docs/iceberg-rest-service.md

```diff
@@ -361,7 +371,7 @@ For example, we can configure Spark catalog options to use Gravitino Iceberg RES
 --conf spark.sql.catalog.rest.uri=http://127.0.0.1:9001/iceberg/
 ```
 
-You may need to adjust the Iceberg Spark runtime jar file name according to the real version number in your environment. If you want to access the data stored in cloud, you need to download corresponding jars (please refer to the cloud storage part) and place it in the classpath of Spark, no extra config is needed because related properties is transferred from Iceberg REST server to Iceberg REST client automatically.
+You may need to adjust the Iceberg Spark runtime jar file name according to the real version number in your environment. If you want to access the data stored in cloud, you need to download corresponding jars (please refer to the cloud storage part) and place it in the classpath of Spark, no extra config is needed if you enable credential vending in the server side by setting `credential-provider-type` and enable credential vending request in client side by setting `spark.sql.catalog.rest.header.X-Iceberg-Access-Delegation` = `vended-credentials`.
```

Comment:
This sentence is too long, you should break it into several short sentences.

Reply:
updated